ChatApp/ChatViewController/ChatViewController.swift

```diff
@@ -77,8 +108,39 @@ class ChatViewController: UIViewController {
 
 //MARK: - Switcher Delegate
 extension ChatViewController: SwitcherDelegate {
-    func switcherDidTap() {
-        print("tapped")
+    func switcherDidTap(_ state: SwitcherState) {
+        switch state {
+        case .light:
+            setUpSwitcherMode()
+        case .dark:
+            setUpSwitcherMode()
+        }
+        
+        func setUpSwitcherMode() {
+            if state == .light {
+                view.backgroundColor = .systemBackground
+                topChatView.typingAreaView.messageTextView.textColor = .black
+                bottomChatView.typingAreaView.messageTextView.textColor = .black
+                statusBar = .darkContent
+            } else {
+                view.backgroundColor = Constants.Color.darkModeBackgroundColor
+                topChatView.typingAreaView.messageTextView.textColor = .white
+                bottomChatView.typingAreaView.messageTextView.textColor = .white
+                statusBar = .lightContent
+            }
+            self.setNeedsStatusBarAppearanceUpdate()
+        }
+    }
+}
+
+//MARK: - Send Delegate
+extension ChatViewController: ChatViewDelegate {
+    func send(message: MessageEntity, fromTop: Bool) {
+        if fromTop == true {
```

Comment:
fromTop == true   =>  fromTop

no need to use '== true' since 'fromTop' is not optional